script.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+class Todo {
+  constructor() {
+    this.toDos = [];
+    this.nextId = 0;
+  }
+  addItem(title, priority) {
+    const toDoItem = {
+      id = this.nextId,
+      title,
+      priority
+    }
+    this.toDos.push(toDoItem);
+    this.nextId++;
+    return toDoItem.id;
+  }
+  removeItem(idToDelete) {
+      if(this.toDos.length) {
+        let indexToDelete = 0;
+        for (let obj of this.toDos) {
```

Comment:
Why do you need this loop? just find an element with id
Also, do not use for loop with arrays
Will be better to name variable todo instead of obj - it is too general